pkg/controller/assignmetadata/assignmetadata_controller.go

```diff
@@ -120,6 +132,34 @@ type AssignMetadataReconciler struct {
 // and what is in the constraint.Spec
 func (r *AssignMetadataReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Info("Reconcile", "request", request)
-
+	deleted := false
+	m := &mutationsv1alpha1.AssignMetadata{}
+	err := r.Get(context.TODO(), request.NamespacedName, m)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return reconcile.Result{}, err
+		}
+		deleted = true
+		// be sure we are using a blank constraint template so that
+		// we know finalizer removal code won't break (can be removed once that
+		// code is removed)
+		m = &mutationsv1alpha1.AssignMetadata{}
+	}
+	deleted = deleted || !m.GetDeletionTimestamp().IsZero()
+
+	if !deleted {
+		log.Info("FEDE", "Inserting in cache")
```

Comment:
This will fail, Info requires an even number of parameters

Reply:
Also, it's not good to leave a comment with my name :p